pkg/middleware/api_key.go

```diff
@@ -53,20 +53,20 @@ func ApiKey(apiKeyRepository repositories.ApiKeyRepository) echo.MiddlewareFunc
 	}
 }
 
-func tryGetApiKeyHeader(req *http.Request) (apiKey uuid.UUID, err error) {
+func TryGetApiKeyHeader(req *http.Request) (apiKey uuid.UUID, err error) {
```

Comment:
Same as above, this is used in the `auth` endpoint to fetch the API key so it has to be public now.